source/codegen/validate_examples.py

```diff
@@ -59,18 +59,16 @@ def _validate_examples(
     print(f"Validating examples using staging_dir: {staging_dir}")
 
     with _create_stage_dir(staging_dir):
-        _system("poetry new .")
-        _system("poetry add grpcio")
-        _system("poetry add --dev grpcio-tools mypy mypy-protobuf types-protobuf grpc-stubs")
-        _system("poetry add --dev black==23.3.0")
-        _system("poetry install")
-
         _stage_client_files(artifact_location, staging_dir)
         examples_dir = staging_dir / "examples"
         proto_dir = staging_dir / "proto"
 
         proto_files_str = str.join(" ", [file.name for file in proto_dir.glob("*.proto")])
 
+        pyproject_file_path = examples_dir / "pyproject.toml"
```

Comment:
I think with your addition of the "stage_client_files" that is doing the copying of these files now so these to lines are no longer needed before the "poetry install" run.